arrow-docs/docs/iso/README.md

```diff
@@ -123,26 +99,28 @@ The `Iso` will be generated as a extension property on the companion object `val
 }
 ```
 ```kotlin:ank:silent
-val iso: Iso<Pos, Tuple2<Int, Int>> = Pos.iso
+val iso: Iso<Pos, Pair<Int, Int>> = Pos.iso
 ```
 
 ### Polymorphic Isos
 When dealing with polymorphic equivalent structures, we can create polymorphic Isos allowing us to morph the type of the focus (and, as a result, the constructed type) of our `PIso`.
 
-Given our previous structures `Tuple2<A, B>` and `Pair<A, B>`, we can create a polymorphic `PIso` that represents a `get: (Tuple2<A, B>) -> Pair<A, B>` and a `reverseGet: (Tuple2<C, D) -> Pair<C, D>`.
+Given our previous structures `Pair<A, B>` and a structure `Tuple2<A, B>`, we can create a polymorphic `PIso` that represents a `get: (Pair<A, B>) -> Tuple2<A, B>` and a `reverseGet: (Tuple2<C, D) -> Pair<C, D>`.
 
-```kotlin
-fun <A, B, C, D> tuple2(): PIso<Tuple2<A, B>, Pair<C, D>, Pair<A, B>, Tuple2<C, D>> = PIso(
-        { tuple -> tuple.a to tuple.b },
-        { tuple -> tuple.a to tuple.b }
+```kotlin:ank
+data class Tuple2<A, B>(val a: A, val b: B)
+
+fun <A, B, C, D> pair(): PIso<Pair<A, B>, Pair<C, D>, Tuple2<A, B>, Tuple2<C, D>> = PIso(
+  { (a, b) -> Tuple2(a, b) },
+  { (a, b) -> a to b }
 )
 ```
 
 `PIso` (defined above) can lift a `reverse` function of `(Pair<A, B>) -> Tuple2<B, A>` to a function `(Tuple2<A, B>) -> Pair<B, A>`.
 
-```kotlin
+```kotlin:ank
 val reverseTupleAsPair: (Tuple2<Int, String>) -> Pair<String, Int> =
-        tuple2<Int, String, String, Int>().lift { intStringPair -> intStringPair.second toT intStringPair.first }
+        pair<Int, String, String, Int>().lift { intStringPair -> intStringPair.second toT intStringPair.first }
```

Comment:
Is`toT` method still available in `0.13` version?

Reply:
Actually I don't think it will but that reminds me that we have to deprecate it 😓 I created a ticket for it 🕵️

Reply:
Done